syntax = "proto3";

import "ml2/tools/protos/ltl.proto";

service Spot {
    rpc CheckSat(LTLSatProblem) returns (LTLSatSolution) {}
    rpc MCTrace(LTLTraceMCProblem) returns (TraceMCSolution) {}
}